### ISSUE TITLE
Control de servos

### ISSUE THREAD

**Issue by David-Estevez** [asrob-uc3m/hexakiller#3]
El hexápodo tiene una barbaridad de servos (mínimo 18) que necesitarán control. Un Arduino normal en principio no puede manejar tantos, así que esta issue servirá para decidir un método de control alternativo. 

De momento, tenemos los siguientes:

* Arduino Mega (propuesto por @davidvelascogarcia )
* [Placa de control de Servos, 16 canales](https://es.aliexpress.com/item/Servo-motor-controller-Driver-Board-16-channels-PC-software-APP-Robot-Serial-port-IIC-Bluetooth-Analog/32610076574.html) (habría que meter 2 de estas):
![image](https://user-images.githubusercontent.com/1451819/33673141-495721c8-daac-11e7-8153-996571244f95.png)
* [Placa de control de Servos, 32 canales](https://es.aliexpress.com/store/product/retail-or-wholesale-Double-shaft-Metal-Gear-robot-Servo-25kg-Motor-for-Arduino-DIY-Robot-servo/517382_32368794229.html)
![image](https://user-images.githubusercontent.com/1451819/33673214-7c6d4e02-daac-11e7-8fd6-8ea88e52dfcb.png)
* [Placa de control de Servos, 32 canales y diversas funciones extra](https://es.aliexpress.com/store/product/32-Channel-Robot-Servo-Control-Board-controller-robot-control-PS2-remote-control-mini-usb-bluetooth-module/2388077_32722390676.html)
![image](https://user-images.githubusercontent.com/1451819/33673242-8e1c036e-daac-11e7-9f30-9cee5dfcca68.png)

**Comment by davidvelascogarcia** [comment]
@David-Estevez A la vista de los acontecimientos de hoy, cambio mi propuesta :), una placa de control de servos de 32 canales, nos facilitaría el conexionado quitando cables. Para la diferencia tan pequeña la de 32 vendría mejor para el caso de adaptar la torreta sobre el hexápodo.

**Comment by David-Estevez** [comment]
El único problema que le veo es que nosotros habíamos pensado en dividir la alimentación de los servos en dos baterías, y con una sola controladora de 32 servos no tenemos esa opción...

**Comment by davidvelascogarcia** [comment]
Cierto, en ese caso si que harían falta dos placas de 16 canales.

**Comment by David-Estevez** [comment]
Hay otra alternativa más: diseñar nuestra propia controladora... pero igual es un marrón en el que no nos deberíamos meter, al menos en un primer prototipo. Cuando todo funcione, se puede cambiar todo por una electrónica custom, pero sin haber probado nada antes es añadir un elemento más que puede fallar...

**Comment by davidvelascogarcia** [comment]
Estoy de acuerdo, una vez sea funcional, se podría mirar preparar algo de electrónica.

**Comment by David-Estevez** [comment]
La placa que estoy usando finalmente es la basada en el PCA9685.

**Comment by David-Estevez** [comment]
https://github.com/edlins/libPCA9685
https://learn.adafruit.com/raspberry-pi-wifi-controlled-cat-laser-toy/hardware
